cucumber/policy/features/deletion.feature

```diff
@@ -264,3 +270,36 @@ Feature: Deleting objects and relationships.
     Then the role list includes host "host-01"
     And the role list includes host "host-02"
     And the role list includes host "host-03"
+
+    @smoke
+    Scenario: Privileges removed on conflicting policy statements
```

Comment:
I think @micahlee covered most everything. My only comment is that it might be good to add a similar test case for `!delete` e.g.
```
    - !policy
      id: test
      body:
      - !delete
        record: !variable db-password
      - !variable db-password
 ```

Reply:
@doodlesbykumbi added the !delete case into the test case per your request.
Please review